invenio_vocabularies/resources/__init__.py

```diff
@@ -9,7 +9,14 @@
 """Resources module."""
 from invenio_vocabularies.resources.schema import L10NString, VocabularyL10Schema
 
+from .config import VocabulariesResourceConfig, VocabularyTypeResourceConfig
+from .resource import VocabulariesAdminResource, VocabulariesResource
```

Comment:
Nice, we should try more to expose objects at the top level of the sub-module namespaces.